docs/pages/server-access/rbac.mdx

```diff
@@ -52,10 +52,10 @@ spec:
       # the list example above can be expressed as:
       'reg': '^us-west-1|eu-central-1$'
 
+    # Options specific to users create by Teleport via the create_host_user role option
```

Comment:
`create_host_user` is obsolete. The new option is `create_host_user_mode` that supports keeping the user instead of deleting it.